tests/console/year_2038_detection.pm

```diff
@@ -25,15 +25,11 @@ use version_utils qw(is_transactional is_sle is_sle_micro is_leap is_leap_micro)
 use Utils::Systemd 'disable_and_stop_service';
 use power_action_utils 'power_action';
 use Utils::Backends 'is_pvm';
+use wrapper;
 
 sub install_pkg {
-    if (is_transactional) {
-        trup_call('pkg install chrony');
-        check_reboot_changes;
-    }
-    else {
-        zypper_call('in chrony');
-    }
+    install_package(packages => 'chrony');
+    check_reboot_changes if is_transactional;
```

Comment:
This looks like a case for repeating pattern. What about to put `check_reboot_changes` to `install_package` and  add new optional  parameter `trup_check_reboot =>  1`  ?

Reply:
Done :+1: